lib/Completion/Tests/Integration/Bridge/TolerantParser/ReferenceFinder/ExpressionNameCompletorTest.php

```diff
@@ -155,19 +155,30 @@ function (Suggestions $suggestions): void {
                 NameSearchResult::create('class', 'Class'),
                 NameSearchResult::create('constant', 'FOO'),
             ],
-            '<?php class Foobar { private const FOO; private const BAR = FOO<>  }',
+            '<?php class Foobar { private const FOO = 1; private const BAR = FOO<>  }',
```

Comment:
why were these test cases updated? did this PR change their behavior?

Reply:
Not related to this PR. If you want, I can move it to a separate change.

Reply:
no that's fine :+1:

Reply:
It was a syntax error not caught by Tolerant Parser in tests. The change did not affect any test result so I allowed myself to leave it here.